drivers/hmis/app/graphql/schema.graphql

```diff
@@ -4577,6 +4578,60 @@ enum RecordType {
   SSVF_SERVICE
 }
 
+type ReferralPosting {
+  assignedDate: ISO8601Date!
+  chronic: Boolean
+  denialNote: String
+  denialReason: String
+  hohName: String!
+  householdSize: Int!
+  id: ID!
+  needsWheelchairAccessibleUnit: Boolean
+  postingIdentifier: ID!
+  referralDate: ISO8601Date!
+  referralIdentifier: ID
+  referralNotes: String
+  referralRequest: ReferralRequest
+  referralResult: ReferralResult
+  referredBy: String!
+  referredFrom: String!
+  resourceCoordinatorNotes: String
+  score: Int
+  status: ReferralPostingStatus!
+  statusNote: String
+  statusNoteUpdatedAt: ISO8601Date
+  statusNoteUpdatedBy: String
+  statusUpdatedAt: ISO8601Date
+  statusUpdatedBy: String
+}
+
+"""
+Referral Posting Status
+"""
+enum ReferralPostingStatus {
+  accepted_by_other_program_status
+  accepted_pending_status
+  accepted_status
+  assigned_status
+  assigned_to_other_program_status
+  closed_status
+  denied_pending_status
+  denied_status
+  new_status
+  not_selected_status
+  void_status
+}
```

Comment:
These are missing descriptions, which we need. The frontend codegen write the descriptions into [this file](https://github.com/greenriver/hmis-frontend/blob/release-77/src/types/gqlEnums.ts) which we use to display the values.

It looks like you generated the enum a bit differently from how we do it elsewhere, which is fine, but we want the end-result to look the same. Looks like you can just modify the generation function in `ReferralPostingStatus`.


An enum following our current pattern would look like the below. It doesn't really matter what the enum value is, but the convention we have is to use uppercase.
```
enum BedType {
  """
  Accepted
  """
  ACCEPTED

  """
  Accepted Pending
  """
  ACCEPTED_PENDING

... etc
}
```

Reply:
Oh I see, you are generating the display value on the frontend https://github.com/greenriver/hmis-frontend/pull/287/files#r1210206980. I still think it's nicer if the GraphQL schema descriptions hold the display strings.

Reply:
Thanks Gig, I'll adjust it.

My understanding is that graphql descriptions are intended for API documentation; I'm not sure that they are supposed to be user-facing. This may be a concern in the future

Reply:
It seems convenient to use them this way and it might be totally fine; it's just a bit unexpected. I could see it being an issue if we ever need to add translations